stdlib/flux_test.go

```diff
@@ -0,0 +1,95 @@
+package stdlib_test
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"testing"
+	"time"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/querytest"
+	"github.com/influxdata/flux/stdlib"
+)
+
+func init() {
+	flux.FinalizeBuiltIns()
+}
+
+var querier = querytest.NewQuerier()
+
+func TestFluxEndToEnd(t *testing.T) {
+	runEndToEnd(t, querier, stdlib.FluxTestPackages)
+}
+func BenchmarkFluxEndToEnd(b *testing.B) {
+	benchEndToEnd(b, querier, stdlib.FluxTestPackages)
+}
+
+func runEndToEnd(t *testing.T, querier *querytest.Querier, pkgs []*ast.Package) {
+	for _, pkg := range pkgs {
+		name := pkg.Files[0].Name
```

Comment:
Should this be the package name not the file name?

Reply:
There can be many test files in the same test package so the file name is probably a better test name (this is the current behavior). Otherwise each test would have the same name (the package name) followed by an index number.

Reply:
I see, I had it backwards. Because we are artificially making each test file its own package (but all with the same name), each test package only has one file, therefore the file name is the better test name.  Makes sense.